BuffettCodeAPIClient/ApiRequestCacheHelper.cs

```diff
@@ -6,19 +6,21 @@ namespace BuffettCodeAPIClient
     public class ApiRequestCacheHelper
     {
         private readonly MemoryCache cache;
+        private readonly string baseUrl;
         private readonly CacheItemPolicy policy = new CacheItemPolicy();
 
-        public ApiRequestCacheHelper(MemoryCache memoryCache)
+        public ApiRequestCacheHelper(MemoryCache memoryCache, string baseUrl)
```

Comment:
ちょっと細かい指摘だけど、呼び出し元で `ToString` するよりもこっちの方が自然な気がした。ApiのRequestのためのクラスだし、結局このクラスが想定してるのはパラメータ名のとおり `baseUrl` なので。他の意図しない文字列がくる可能性も減らせるし。

```suggestion
        public ApiRequestCacheHelper(MemoryCache memoryCache, Uri baseUrl)
```

Reply:
ここ迷ったんだよね、cache の key になるのは string だしなぁと思って。ま、でも `Uri` のが自然だね